### PR TITLE
remove Travis caching to get around npm ls issue

### PR DESCRIPTION
**Pull request by rgbkrk**
`master` is currently failing after having the cache busted. It fails when Travis (automatically) runs `npm ls`. I'm going to see if disabling the cache helps. ¯\\\_(ツ)\_/¯

### CODE REVIEW

**Comment by rgbkrk** [comment]
On a separate note, I configured my remotes wrong apparently. Oops. Stuck waiting on multiple Travis builds now...

```
origin	git@github.com:nteract/nteract.git (fetch)
origin	git@github.com:nteract/nteract.git (push)
rgbkrk	git@github.com:nteract/nteract.git (fetch)
rgbkrk	git@github.com:nteract/nteract.git (push)
```

File: .travis.yml [hunk 1]
```diff
@@ -14,11 +14,6 @@ env:
   - GROUP=flow
   - GROUP=conformance
 
-cache:
-  directories:
-    - $HOME/.npm
-    - node_modules
```
Comment (lgeiger) [inline .travis.yml]: It may be enough to remove only one of the folders

**Review by lgeiger — APPROVED** [review]
:+1: let's wait what Travis thinks

**Comment by rgbkrk** [comment]
It's looking like the issue stems with something in `ajv` (which we don't directly depend on). `nbschema` only exports the literal schema and we use `jsonschema` for validation in the conformance tests.

**Comment by rgbkrk** [comment]
Oh whoops, now that I've changed remotes I'm not pushing to the right branch.

_tries again_

**Comment by rgbkrk** [comment]
`/me whistles`

I'll turn caching back on and resolve to the fact that it's `npm ls` that isn't working (likely with our lerna setup).

**Comment by rgbkrk** [comment]
cache back guarantee

**Review by peggyrayzis — APPROVED** [review]
🎉

**Comment by rgbkrk** [comment]
Thank you!

**Pull request merged**